sql/ddl_api.sql

```diff
@@ -17,23 +17,26 @@ CREATE OR REPLACE FUNCTION  create_hypertable(
     number_partitions       INTEGER = NULL,
     associated_schema_name  NAME = NULL,
     associated_table_prefix NAME = NULL,
-    chunk_time_interval     BIGINT =  _timescaledb_internal.interval_to_usec('1 month'),
+    chunk_time_interval     BIGINT = NULL,
```

Comment:
Since we can overload, I like this idea potentially. I think it should be a separate PR just to keep this streamline.